File: sentimental-mario-less/mario.py
from cs50 import get_int

def main():
    hight = get_hight()
    for j in range(hight):

        for k in range(hight-j-1):
            print(" ", end="")

        for l in range(j+1):
            print("#", end="")

        print()

def get_hight():
    i = 0
    while 1 > i or i > 8:
        i = get_int("hight: ")
    else:
        return i
main()

File: mario-less/mario.c
#include <cs50.h>
#include <stdio.h>

int main(void)
{
    //get int from user
    int hight;
    do
    {
        hight = get_int("Hight of the pyramid between 1 - 8? ");
    }
    while (1 > hight || hight > 8);

    //print pyramid

    for (int i = 0; i < hight; i++)
    {
        for (int k = hight - i - 1; k > 0; k--)
        {
            printf(" ");
        }
            for (int j = 0; j <= i; j++)
            {
                printf("#");
            }
        printf("\n");
    }
}